File: Assignments/2/A3_AKHIL_CS19B053/3.2/tworay_parabolic_1.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
double Pt=50.00;
double d=1000.00;
double Pi=2*asin(1);
double lamda=(double)(3/(double)9);
vector<double>v;
for(double Ht=10;Ht<=50;Ht+=10){
    for(double Hr=1;Hr<=5;Hr++){
    double Pr;
    double Gain;
    Gain=10*log10((0.6*(pow(((Pi)*3)/lamda,2))));
    Pr=(double)(Pt)*(pow(Ht*Hr,2)*Gain)/pow(d,4);
    Pr=30+10*log10(Pr);
    v.push_back(Pr);
    }
}
int freq2=100;
int k=0;
for(double Ht=10;Ht<=50;Ht+=10){
    for(double Hr=1;Hr<=5;Hr++){
        cout<<Ht<<" "<<Hr<<" "<<v[k]<<endl;
        k++;
    }
}
}

File: Assignments/2/A3_NUTHANNAIK_CS19B051/ques2/tworay_parabolic_1.py
import math

pi = 2*math.asin(1)

def function():
	pt = 50.00
	d = 1000.00
	lamba = 3.00/9.00
	v = []
	for hr in range(10, 60, 10):
		for ht in range(1, 6, 1):
			gain = 10* math.log10(0.6*(((pi*3)/lamba)**2))
			pr = ((pt *((hr*ht)**2))*gain)/(d**4)
			pr = 30 + 10*(math.log10(pr))
			v.append(pr)
	k = 0
	for i in range(1, 6):
		j = i*10
		for hr in range(1, 6):
			print(str(j) + " " + str(hr) + " " + str(v[k]))
			k += 1
	print()

if __name__ == '__main__':
	function()